src/vm/builtins/builtin_runner/output.zig

```diff
@@ -12,6 +58,28 @@ pub const OutputBuiltinRunner = struct {
     stop_ptr: ?usize,
     /// Included boolean flag
     included: bool,
+    /// A mapping from page IDs to their respective PublicMemoryPage configurations.
+    ///
+    /// This map stores associations between page IDs and their corresponding PublicMemoryPage configurations.
+    pages: AutoHashMap(usize, PublicMemoryPage),
+
+    /// Initializes a new instance of the OutputBuiltinRunner.
+    ///
+    /// This function creates and returns a new OutputBuiltinRunner instance
+    /// with default configuration, setting the base address to 0, stop pointer to null,
+    /// and marks the runner as included by default.
+    ///
+    /// # Returns
+    ///
+    /// A new instance of OutputBuiltinRunner with default settings.
+    pub fn init(allocator: Allocator) Self {
```

Comment:
can this be `initDefault`? zig precedence for this:

https://github.com/ziglang/zig/blob/master/lib/std/enums.zig#L737

Reply:
Agreed done